public/app/features/alerting/unified/components/rule-editor/QueryRows.tsx

```diff
@@ -278,6 +285,19 @@ function copyModel(item: AlertQuery, uid: string): Omit<AlertQuery, 'datasource'
   };
 }
 
+function emptyModel(item: AlertQuery, uid: string): Omit<AlertQuery, 'datasource'> {
+  return {
+    refId: item.refId,
+    relativeTimeRange: item.relativeTimeRange,
+    queryType: '',
+    datasourceUid: uid,
+    model: {
+      refId: item.refId,
```

Comment:
While working on adding support for defining a default query, I wasn't aware unified alerting had it's own query rows renderer. That means default queries defined by the data source will be ignored in the alerting ui. 

If you want to, you spread the default query to the alert query model here.
```typescript
function emptyModel(
  item: AlertQuery,
  ds: DataSourceApi<DataQuery, DataSourceJsonData, {}>
): Omit<AlertQuery, 'datasource'> {
  return {
    refId: item.refId,
    relativeTimeRange: item.relativeTimeRange,
    queryType: '',
    datasourceUid: ds.uid,
    model: {
      ...ds?.getDefaultQuery?.(CoreApp.UnifiedAlerting),
      refId: item.refId,
      hide: false,
    },
  };
}
```

Maybe then this function should be called `newModel` rather than `emptyModel`. 

But let me know if you want me to do this in a separate PR.

Reply:
Ah, this is great! i will apply your suggestion. Thanks 🙏🏻